Vagrantfile

```diff
@@ -54,23 +77,18 @@ Vagrant.configure("2") do |config|
     echo "."
 
     if [ -n "$HTTP_PROXY" ] || [ -n "$HTTPS_PROXY" ]; then
-      bash <(curl -L -s --proxy ${HTTP_PROXY} ${CODENVY_URL}) ${CODENVY_OPTIONS} --http-proxy=${HTTP_PROXY} --https-proxy=${HTTPS_PROXY}
+      bash <(curl -L -s --proxy ${HTTP_PROXY} ${CODENVY_URL}) ${CODENVY_OPTIONS} \
+                                                              --http-proxy-for-installation=${HTTP_PROXY} \
+                                                              --https-proxy-for-installation=${HTTPS_PROXY}
     else
       bash <(curl -L -s ${CODENVY_URL}) ${CODENVY_OPTIONS}
     fi
 
     echo "."
     echo "."
-    echo "CODENVY: DOWNLOADING POPULAR STACKS"
-    echo "."
-    echo "."
-    docker pull codenvy/ubuntu_jdk8
-
-    echo "."
-    echo "."
     echo "CODENVY: INSTALLED!"
-    echo 'Add "192.168.56.110 codenvy" to your hosts file'
-    echo 'Access:   http://codenvy'
+    echo 'Add       "192.68.56.110 codenvy.onprem" to your hosts file'
```

Comment:
Correct default IP address is *192.168.56.110* here. Moreover, *$ip* is more preferable here so as there could **$CODENVY_IP** be used instead of default one, according to the command above:
`ip              = ENV['CODENVY_IP'] || "192.168.56.110"`